app/styles/components/advertisement-tool.scss

```
@@ -1,5 +1,5 @@
 // CSS3 code for animation of advertisements
-$amount-of-advertisements: 14;
+$amount-of-advertisements: 19;
```

If you replace this with 18, the animation where it restarts at the first ad is smoother
```suggestion
$amount-of-advertisements: 18;
```